go.mod

```diff
@@ -1,13 +1,54 @@
 module github.com/openshift/coredns-mdns
 
 require (
-	github.com/caddyserver/caddy v1.0.4
+	cloud.google.com/go v0.78.0 // indirect
+	github.com/Azure/azure-sdk-for-go v51.3.0+incompatible // indirect
+	github.com/Azure/go-autorest/autorest v0.11.18 // indirect
+	github.com/Azure/go-autorest/autorest/azure/auth v0.5.7 // indirect
+	github.com/Azure/go-autorest/autorest/to v0.4.0 // indirect
+	github.com/DataDog/datadog-go v4.4.0+incompatible // indirect
+	github.com/Microsoft/go-winio v0.4.16 // indirect
+	github.com/Shopify/sarama v1.21.0 // indirect
+	github.com/aws/aws-sdk-go v1.37.16 // indirect
+	github.com/caddyserver/caddy v1.0.4 // indirect
 	github.com/celebdor/zeroconf v0.0.0-20190404095836-c328d57fca11
 	github.com/cenkalti/backoff v2.2.1+incompatible // indirect
-	github.com/coredns/coredns v1.6.6
-	github.com/miekg/dns v1.1.25
+	github.com/coredns/caddy v1.1.0
+	github.com/coredns/coredns v1.8.1
+	github.com/coredns/federation v0.0.0-20190818181423-e032b096babe // indirect
+	github.com/coreos/etcd v3.3.25+incompatible // indirect
+	github.com/coreos/go-semver v0.3.0 // indirect
+	github.com/coreos/go-systemd v0.0.0-20191104093116-d3cd4ed1dbcf // indirect
+	github.com/coreos/go-systemd/v22 v22.1.0 // indirect
+	github.com/dnstap/golang-dnstap v0.4.0 // indirect
+	github.com/gogo/protobuf v1.3.2 // indirect
+	github.com/google/gofuzz v1.2.0 // indirect
+	github.com/google/uuid v1.2.0 // indirect
+	github.com/googleapis/gnostic v0.5.4 // indirect
+	github.com/gophercloud/gophercloud v0.15.0 // indirect
+	github.com/imdario/mergo v0.3.11 // indirect
+	github.com/infobloxopen/go-trees v0.0.0-20200715205103-96a057b8dfb9 // indirect
+	github.com/miekg/dns v1.1.38
 	github.com/openshift/mdns-publisher v0.0.0-20200219161514-0e41fab1ddec
-	golang.org/x/net v0.0.0-20191209160850-c0dbc17a3553
+	github.com/openzipkin/zipkin-go v0.2.5 // indirect
+	github.com/prometheus/common v0.17.0 // indirect
+	github.com/prometheus/procfs v0.6.0 // indirect
+	github.com/tinylib/msgp v1.1.5 // indirect
+	go.etcd.io/etcd v3.3.25+incompatible // indirect
+	go.opencensus.io v0.22.6 // indirect
+	go.uber.org/multierr v1.6.0 // indirect
+	go.uber.org/zap v1.16.0 // indirect
+	golang.org/x/crypto v0.0.0-20210220033148-5ea612d1eb83 // indirect
+	golang.org/x/net v0.0.0-20210222171744-9060382bd457
+	golang.org/x/oauth2 v0.0.0-20210220000619-9bb904979d93 // indirect
+	golang.org/x/term v0.0.0-20210220032956-6a3ed077a48d // indirect
+	golang.org/x/time v0.0.0-20210220033141-f8bda1e9f3ba // indirect
+	gopkg.in/yaml.v2 v2.4.0 // indirect
+	k8s.io/api v0.20.4 // indirect
+	k8s.io/client-go v11.0.0+incompatible // indirect
```

Comment:
is this version tag for client-go right?

Reply:
It looks like I passed an errant -u to go get when I updated coredns. I think this should be fixed now.